course_info/data_migrations/canvas_account_admin_perms.sql

```diff
@@ -13,7 +13,7 @@ INSERT INTO lti_permissions_ltipermission (permission, school_id, canvas_role, a
 	('account_admin_tools', '*', 'Librarian', FALSE),
 	('account_admin_tools', '*', 'ObserverEnrollment', FALSE),
 	('account_admin_tools', '*', 'SchoolLiaison', TRUE),
-	('account_admin_tools', '*', 'Shopper', FALSE),
+	('account_admin_tools', '*', 'Prospective Enrollee', FALSE),
```

Comment:
I just took this out entirely, as an absence of a rule defaults to `FALSE`. I think it's the same, really. (I also removed `Harvard-Viewer` too)